app/views/jobs/_heating_element_statuses_dashboard.html.erb

```diff
@@ -0,0 +1,19 @@
+<!--<div class="list-group panel">-->
+<!--<div class="table-responsive">-->
+<!--<h4 class="list-group-item list-group-item-success">Heating Element</h4>-->
+<!--<table id="heating_element_statuses_dashboard" class="heating_element_statuses table table-hover table-striped">-->
+<!--<thead>-->
+<!--<tr>-->
+<!--<th>Device ID</th>-->
+<!--<th>Heating Element ID</th>-->
+<!--<th>State</th>-->
+<!--<th>Designated Stop Time</th>-->
```

Comment:
Since you supply a column for voltage in _heating_element_status.html.erb, you need to supply a column header for Voltage here.